Frameworks/UIKit/XamlControls.h

```diff
@@ -121,6 +121,15 @@ WXCGrid* CreateLabel();
 WXCTextBlock* GetLabelTextBlock(WXCGrid* labelGrid);
 
 ////////////////////////////////////////////////////////////////////////////////////
+// TextField
+////////////////////////////////////////////////////////////////////////////////////
+WXCCanvas* CreateTextField();
+WXCCanvas* GetTextFieldSubLayerCanvas(WXFrameworkElement* textField);
+WXCTextBox* GetTextFieldTextBox(WXFrameworkElement* textField);
+WXCPasswordBox* GetTextFieldPasswordBox(WXFrameworkElement* textField);
+void SetTextFieldSecureTextEntryValue(WXFrameworkElement* textField, bool secureTextEntry);
```

Comment:
actually SecureTextEntry is what reference platform is using for SecureTextMode. so i will keep it.

---
In reply to: [110059600](https://github.com/Microsoft/WinObjC/pull/2411#discussion_r110059600) [](ancestors = 110059600)